codecept.conf.js

```diff
@@ -12,7 +29,7 @@ exports.config = {
       show: process.env.SHOW_BROWSER_WINDOW || false,
       restart: true,
       waitForNavigation: 'networkidle0',
-      waitForTimeout: 30000,
+      waitForTimeout: 15000,
```

Comment:
This is to reduce wait time in E2E tests that are about to fail. 15 seconds should be good as default for local and AKS. Could be extended in the future for slower envs.